src/EnergyPlus/ElectricPowerServiceManager.cc

```diff
@@ -2693,20 +2693,13 @@ DCtoACInverter::DCtoACInverter(EnergyPlusData &state, std::string const &objectN
         if (zoneNum_ > 0) {
             switch (modelType_) {
             case InverterModelType::SimpleConstantEff: {
-                SetupZoneInternalGain(state,
-                                      zoneNum_,
-                                      "ElectricLoadCenter:Inverter:Simple",
-                                      name_,
-                                      DataHeatBalance::IntGainType::ElectricLoadCenterInverterSimple,
-                                      &qdotConvZone_,
-                                      nullptr,
-                                      &qdotRadZone_);
+                SetupZoneInternalGain(
+                    state, zoneNum_, name_, DataHeatBalance::IntGainType::ElectricLoadCenterInverterSimple, &qdotConvZone_, nullptr, &qdotRadZone_);
```

Comment:
And even ended up getting rid of some lines of code thanks for the reduced arguments, nice.